alicat/serial.py

```diff
@@ -268,9 +405,14 @@ def set_flow_rate(self, flow, retries=2):
         Args:
             flow: The target flow rate, in units specified at time of purchase
         """
-        if self.control_point is not None and self.control_point != 'flow':
+        if any([self.control_point is not None and
+                self.control_point == 'abs pressure',
+                self.control_point is not None and
+                self.control_point == 'gauge pressure',
+                self.control_point is not None and
+                self.control_point == 'diff pressure']):
```

Comment:
Hm, how about

```python
if self.control_point in ['abs pressure', 'gauge pressure', 'diff pressure']:
```

Reply:
Good point! For some reason I remember trying this and having trouble with it. Just changed it and no issues with any device.